plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/StandardProjectWizard.java

```diff
@@ -150,30 +149,14 @@ public void init(IWorkbench workbench, IStructuredSelection selection) {
     }
   }
 
-  private static class DependencyValidator implements IRunnableWithProgress {
+  private class DependencyValidator implements IRunnableWithProgress {
 
     private IStatus result = null;
 
     @Override
     public void run(IProgressMonitor monitor) throws InvocationTargetException,
```

Comment:
this line confused me at first; please break before throws instead to make it a bit clearer